lintrunner_adapters/adapters/pip_init.py

```diff
@@ -67,6 +72,8 @@ def run_command(args: list[str]) -> subprocess.CompletedProcess[bytes]:
     # these cases perform a regular installation.
     in_conda = os.environ.get("CONDA_PREFIX") is not None
     in_virtualenv = os.environ.get("VIRTUAL_ENV") is not None
+    if args.force_venv and not in_virtualenv:
+        raise RuntimeError("Activate virtualenv to install packages.")
```

Comment:
Thanks! Could you add something along the lines of “Fails to init because… and force_venv is True” so users know why?